### PR TITLE
[AIRFLOW-1274][HttpSensor] Rename parameter params to data

### PR DESCRIPTION
**Pull request by aliceabe**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW-1274) issue and references it in the PR title.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:

No UI changes, I've just renamed the parameter `params` to `request_params` in `HttpSensor` as it was conflicting with `BaseOperator` `params` and causing failed DAGs parsing.

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
 
They already exist.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue.

@saguziel @aoen

### CODE REVIEW

File: airflow/example_dags/example_http_operator.py [hunk 1]
```diff
@@ -81,7 +81,7 @@
     task_id='http_sensor_check',
     http_conn_id='http_default',
     endpoint='',
-    params={},
```
Comment (aoen) [inline airflow/example_dags/example_http_operator.py:84]: Nit: Can we call these request_params or something instead? I think this is a more logical name.

**Comment by aoen** [comment]
Thank you for making this fix!

File: airflow/example_dags/example_http_operator.py (hunk 1 above)
Comment (aliceabe) [reply airflow/example_dags/example_http_operator.py:84]: Changed

**Comment by aoen** [comment]
LGTM

**Pull request closed without merging**